.github/workflows/build.yaml

```diff
@@ -1,7 +1,25 @@
 name: Build
 
 on:
+  workflow_call:
+    inputs:
+      refLsp4ij:
+        description: 'Reference/branch for Lsp4ij checkout'
+        type: string
+        required: true
+        default: main
+      useLocalPlugin:
+        description: 'Use lsp4ij locally'
+        required: true
+        type: boolean
+        default: false
   workflow_dispatch:
+    inputs:
+      useLocalPlugin:
+        description: 'Use lsp4ij locally'
+        required: true
+        type: boolean
+        default: false
```

Comment:
If we want to be able to manually run builds that target a specific PR, do we need to add the `refLsp4ij` input under `workflow_dispatch` as well? If that is the case, is there a way to specify `inputs` one time which then applies to both `workflow_call` and `workflow_dispatch` (and any other invocation added in the future)?

Reply:
Yes we need to add `refLsp4ij` input under workflow_dispatch to manually run builds that target a specific merge commit SHA. So that we can add this sha input as a string . But i dont think we can avoid duplication of inputs in `workflow_call` and `workflow_dispatch` . Seems like there is no way to shared_inputs between workflow_call and workflow_dispatch as mentioned [here](https://github.com/orgs/community/discussions/39357#discussioncomment-7500641)

Reply:
Added `refLsp4ij` input under `workflow_dispatch`

Reply:
https://stackoverflow.com/a/78683976/22367958 - We cannot avoid duplication of inputs in workflow_call and workflow_dispatch.